src/misc/AttachShadowPolyfill.ts

```diff
@@ -1,8 +1,13 @@
 import { $$ } from '../utils/Dom';
```

Comment:
The changes to `AttachShadowPolyfill` don't fix the issue and aren't necessary. However, due to not knowing when the content is resized, it made it yet another place where the issue *could* happen. I figured I should improve it now to avoid future confusion or even future issues.